lib/util/api-json.js

```diff
@@ -63,7 +63,11 @@ function api(token, endpoint, method, requestBody, callback) {
                 if (responseBody.trim().length === 0) {
                     responseBody = null;
                 } else {
-                    responseBody = JSON.parse(responseBody);
+                    try {
+                        responseBody = JSON.parse(responseBody);
+                    } catch (e) {
+                        return callback(new APIError(e, responseBody, responseCode));
```

Comment:
@nicjansma why `return callback()`?  Other places in this function use just `callback` or `callback && callback()`.